### ISSUE TITLE
Incorrect location shown on website "deploy anywhere" section

### ISSUE THREAD

**Issue by ishanloya** [sveltejs/kit#8054]
### Describe the bug

Until yesterday, the correct location was being shown. The pin still seems to be roughly in the correct location. The label, however reads New York City.

<img width="1106" alt="Screenshot 2022-12-10 at 9 56 04 AM" src="https://user-images.githubusercontent.com/17540310/206828922-f8d15967-2296-4959-8507-ba3c6441278f.png">

### Reproduction

Visit the latest website and scroll to the "deploy anywhere" section

### System Info

```Shell
Tried on multiple browsers on a Mac: Brave, Safari, Firefox
```

### Severity

serious, but I can work around it

**Comment by geoffrich** [comment]
Presumably this is because the `x-vercel-ip-city` header is undefined and the fallback is "New York City". The fix for this is probably just to have a better fallback - either "unknown city" or just leave it blank?

I guess we'd only want the "New York City, USA" fallback if the lat/long are _also_ undefined.

https://github.com/sveltejs/kit/blob/4b4c750c70866d4c38353d7150b8b5105402141a/sites/kit.svelte.dev/src/routes/edge.svg/+server.js#L111

**Comment by Rich-Harris** [comment]
lol, oops 😬 

yeah, that solution makes sense. I think we should also have a country lookup, so that it says 'India' instead of 'IN'

**Comment by Rich-Harris** [comment]
closed via #8067 (forgot to tag this issue)
